src/solution.c

```diff
@@ -112,6 +112,15 @@ void solution_send_nmea(gnss_solution *soln, dops_t *dops,
 
 }
 
+double calc_heading(const double b_ned[3])
+{
+  double heading = atan2(b_ned[1], b_ned[0]);
+  if (heading < 0) {
+    heading += 2 * M_PI;
+  }
+  return heading * (180 / M_PI);
```

Comment:
We have a constant `R2D` for this, nbd but makes it a little bit more readable.

https://github.com/swift-nav/libswiftnav/blob/master/include/libswiftnav/constants.h#L25